src/docfx/Models/InitCommand.cs

```diff
@@ -372,8 +372,12 @@ private class SingleChoiceQuestion<T> : Question<T>
         public SingleChoiceQuestion(string content, Action<T, DefaultConfigModel, QuestionContext> setter, Func<string, T> converter, params string[] options)
             : base(content, setter)
         {
-            if (options == null || options.Length == 0) throw new ArgumentNullException(nameof(options));
-            _converter = converter ?? throw new ArgumentNullException(nameof(converter));
+            ArgumentNullException.ThrowIfNull(options);
+            ArgumentNullException.ThrowIfNull(converter);
+
+            if (options.Length == 0) throw new ArgumentNullException(nameof(options));
```

Comment:
It's intentionally aligned to previous behaviors as described above.
But for consistency. It might better to throw `ArgumentOutOfRangeException` here (and other codes that throw `ArgumentException`).

I'll create PR later.

Reply:
Fixed on commit (https://github.com/dotnet/docfx/pull/9027/commits/7f767448ef85f2c9cd14c5f54906f30e8a895c82).